tests/annotation_test.go

```diff
@@ -43,7 +43,7 @@ func TestAddAnnotation(t *testing.T) {
 	workingDir := setUpTest(t)
 	defer cleanUpTest(workingDir)
 
-	_, _, _, err := runACBuild(workingDir, "annotation", "add", annoName, annoValue)
+	_, _, _, err := runACBuild(workingDir, "--no-history", "annotation", "add", annoName, annoValue)
```

Comment:
are you ever running this with history?!
since 99% of invocations seem to be of the pattern
`_, _, _, err := runACBuild(workingDIr, "--no-history", ...)`
might I suggest a helper func